### DISCUSSION TITLE
How to transfer all private playlists?

### DISCUSSION THREAD

**Discussion by oknchm** [sigma67/spotify_to_ytmusic#145] [Q&A]
I want to transfer all my playlists, but they're all private. Tried the command `spotify_to_ytmusic all <myuserid>` but the result was `0 playlists found. Starting transfer...`. I was able to transfer a few private playlists using the `create` command, but I have a lot. It would be time-consuming to copy them one by one. Please help, thank you.

**Comment by sigma67** [comment]
Did you try oauth setup option for spotify?

**Comment by oknchm** [comment]
Just followed the instructions [here](https://github.com/sigma67/spotify_to_ytmusic#setup) and provided the client id and secret for Spotify. Is it different from OAuth setup?

**Comment by sigma67** [comment]
Did you choose yes or no?

**Comment by oknchm** [comment]
Chose `yes` to use OAuth, even double-checked the `settings.ini` file.

```
...
[spotify]
client_id = [redacted]
client_secret = [redacted]
use_oauth = yes
```

**Comment by baharzaman** [comment]
If you want to transfer all your private playlists (including the ones that aren’t public), tools like Musconvtool can help. It can transfer both private and public playlists between different streaming services, including Spotify, Apple Music, and YouTube Music. You simply connect your accounts in the app, select the playlists you want (including private ones), and transfer them all at once,  so you don’t have to recreate them manually. Saves a lot of time!
